### ISSUE TITLE
[SWAG] T-shirt Purchase Request on behalf of the MPWG

### ISSUE THREAD

**Issue by caitmm43** [microprofile/microprofile-marketing#633]
@croundy and @HailleySeed

Hello, 

Here is the invoice quote for the swag t-shirt order (attached, see PDF) and the link to approve and pay the invoice: 
[https://deluxe.deluxescreenprinting.net/Dsp/BusinessTools/Invoice?ID=23173607&Email=a277f6a5](https://deluxe.deluxescreenprinting.net/Dsp/BusinessTools/Invoice?ID=23173607&Email=a277f6a5)

For additional context, please review the [Dec 6th, 2021 Marketing Minutes](https://docs.google.com/document/d/1fQWUiCInhJni8WhGt2k5WKqb8K6BuhzzWTzqynvX3_U/edit?usp=sharing) + call [Video](https://twitter.com/caitmm43/status/1468761294463541248?s=20), and other documentation included in the drive folder [MP Swag](https://drive.google.com/drive/u/0/folders/17-U4DbUpnYzbiH9qkA6IJyMCK9XxZw36).

Our contact at Deluxe is Elsie Acevedo.  She can be reached via elsie@deluxescreenprinting.net, for any questions.  

We appreciate your part in finalizing this order in time for the EF Dec 14th swag deadline!

Many thanks,
Caitlin 

[12_7_2021_Deluxe MPWG - 380 Shirts_Invoice23173607.pdf](https://github.com/eclipse/microprofile-marketing/files/7695521/12_7_2021_Deluxe.MPWG.-.380.Shirts_Invoice23173607.pdf)

**Comment by HailleySeed** [comment]
Hi @caitmm43.

This has been approved and will be processed for payment. I will get in touch with Elise at Deluxe and let her know that they can proceed with the order and will receive the payment shortly.

Thanks,
Hailley

**Comment by caitmm43** [comment]
Fantastic!  Thank you @HailleySeed

**Comment by croundy** [comment]
Short status update: The supplier is scheduled to start printing next week. However we have asked that they hold off pending the resolution of a couple of issues that have been raised:

1. The use of the word "contributor" on the back of the t-shirt seems a bit odd where it is located. The suggestion from EF is that this be moved to just below the MicroProfile logo and be changes to say something like: "Powered by Contributors"

2. Jelastic has stated that they will no longer be part or the WG. Payara may also not want to be part of this. Please confirm which logos should be included in the design.

Again, the supplier is on hold pending updated artwork. Please update and send at your earliest convenience. Thanks!

**Comment by caitmm43** [comment]
@croundy Hi Clark, Thanks for sharing this update.  We discussed each of these points in great detail before deciding to finalize the design.

> 1. The use of the word "contributor" on the back of the t-shirt seems a bit odd where it is located. The suggestion from EF is that this be moved to just below the MicroProfile logo and be changes to say something like: "Powered by Contributors"
We chose the wording and design that was submitted after discussing an array of alternatives.  We prefer the current version.

> 2. Jelastic has stated that they will no longer be part or the WG. Payara may also not want to be part of this. Please confirm which logos should be included in the design.
Since the t-shirts are being printed with 2021 budget and both Payara and Jelastic remained members in 2021, they must be included on the shirt.  

Our intent is to move forward without further delay.  Please update Deluxe that the current artwork is ready to print.  Thank you!

**Comment by croundy** [comment]
@caitmm43 

Hi Caitlin. 

For point 2, Leaving Payara and Jelastic is fine, so long as both organizations agree to that. 

As for the 1st point, let me rephrase: The foundation will not approve this design with "Contributors" appearing on the t-shirt as if it's another logo. This needs to be re-worked so it is clear that contributors is not a logo. 

Thanks!

> > 1. The use of the word "contributor" on the back of the t-shirt seems a bit odd where it is located. The suggestion from EF is that this be moved to just below the MicroProfile logo and be changes to say something like: "Powered by Contributors"
> >    We chose the wording and design that was submitted after discussing an array of alternatives.  We prefer the current version.
> 
> > 2. Jelastic has stated that they will no longer be part or the WG. Payara may also not want to be part of this. Please confirm which logos should be included in the design.
> >    Since the t-shirts are being printed with 2021 budget and both Payara and Jelastic remained members in 2021, they must be included on the shirt.
> 
> Our intent is to move forward without further delay. Please update Deluxe that the current artwork is ready to print. Thank you!

**Comment by shanda-eclipse** [comment]
Reaching out to let you know Hailley, who was coordinating this project, has left the Eclipse Foundation. I will be continuing the relationship with the supplier when the artwork is finalized.

**Comment by SharonCorbett** [comment]
The Eclipse Foundation requires an update to Clark's request 18 days ago.
As far as we are aware the artwork has not been updated with Delux??
Please remember, this request is in compliance with the Eclipse Foundation Trademark Policy! 
Please respond ASAP.

Thanks,
Sharon

**Comment by SharonCorbett** [comment]
cc @rstjames

**Comment by caitmm43** [comment]
@shanda-eclipse, thanks for introducing yourself!  @croundy and @SharonCorbett, thank you for continuing to move this thread forward.  I am catching back up to speed after a few weeks away from MP.  I've reached out to @rstjames and will follow up here.

**Comment by rstjames** [comment]
@caitmm43 I updated https://github.com/eclipse/microprofile-marketing/issues/595 13 days ago informing all design files were updated on GD. There is a 2021 print shirt and hoodie file and a 2022

**Comment by shanda-eclipse** [comment]
Thanks @rstjames. Can you confirm that we are to use the 2022 files for both the t-shirt and the hoodie?

**Comment by rstjames** [comment]
@shanda-eclipse Last I heard we should use the 2021 version with Jelastic and Piyara in it because these shirts are printed from the 2021 budget. @aeiras is this correct?

**Comment by caitmm43** [comment]
@rstjames Thanks for updating the files and keeping this thread moving.  

@shanda-eclipse It is also my understanding that we need to use the 2021 version, as this order is using 2021's allocated budget.  In my last conversation with Amelia she confirmed the same.  Unless something has changed in the last few weeks, we should now be ready to print.

Let's discuss on the marketing call today to confirm that we are on the same page and make a plan to move forward.

**Comment by caitmm43** [comment]
@shanda-eclipse We discussed this during our call today and confirmed that we are ready to print this order using the updated 2021 designs.  Do you need any additional info from us in order to finalize this order with the printer?

**Comment by shanda-eclipse** [comment]
No further info needed on my side, I will work with the supplier to finalize the order.

**Comment by caitmm43** [comment]
@shanda-eclipse Awesome, thank you!

**Comment by caitmm43** [comment]
Update from Elsie at Deluxe: 

Funds from EF are in transit (there was an initial problem with the first check, it has since been reissued & deposited).  Once the funds clear, they'll add our order to their printing schedule & provide an ETA for completion. 

When the order is complete, we will need to arrange pick up or shipping.  The printer is located in Los Angeles, CA.  @aeiras, how have we done this with past orders?

**Comment by shanda-eclipse** [comment]
FYI - Artwork proofs have been approved. 

![N L 6210 MICROPROFILE BACK](https://user-images.githubusercontent.com/61751191/161597786-37924b02-6b3f-4bee-bfe3-e1fc31efdeef.jpg)
![N L 6210 MICROPROFILE FRONT](https://user-images.githubusercontent.com/61751191/161598039-643e44f5-1356-4db7-a6cd-6860ea0012cd.jpg)

**Comment by edbratt** [comment]
Once payment details are confirmed, shirts will be printed.

**Comment by shanda-eclipse** [comment]
> When the order is complete, we will need to arrange pick up or shipping. The printer is located in Los Angeles, CA. @aeiras, how have we done this with past orders?

@caitmm43 The shirts have been printed. Has a decision been made about shipping/distribution plans? We have the option to bulk ship them to one location or distribute them to a few different groups but the cost of shipping will depend on what we decide to do.

**Comment by shanda-eclipse** [comment]
@SharonCorbett @croundy do either of you have insights into the shipping plans? Deluxe seems urgent to move these.

**Comment by croundy** [comment]
@caitmm43 @edbratt The supplier would like to get these out of their facility. We will have them shipped to designated member sites (preferably no more than 4-5 sites) and let you distribute from there. Can you please let us know how you would like to distribute these t-shirts amongst the members by this Friday, April 29th?  Thanks!

**Comment by caitmm43** [comment]
@shanda-eclipse @croundy @edbratt, 

I believe there are some mis-stated expectations.  From Elsie at Deluxe on April 22nd: 
> "There is no urgency on our part. I can store the tees as long as you need." 

We (mktg group) will solidifying a distribution plan at today's mktg meeting.  Our goal is for Deluxe to ship the t-shirts domestically and then we will orchestrate any international shipping after they have been received (similar to the plan you laid out above). 

Thanks again to all for your followup and continuing to move this order forward!
